src/components/container/SurveyDataChart/SurveyDataChart.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useState, useContext } from 'react'
+import { DateRangeContext } from '../../presentational/DateRangeCoordinator/DateRangeCoordinator'
+import { add, parseISO } from 'date-fns'
+import { SurveyAnswer } from '@careevolution/mydatahelps-js'
+import { WeekStartsOn, getMonthStart, getWeekStart } from '../../../helpers/get-interval-start'
+import TimeSeriesChart, { AreaChartOptions, BarChartOptions, LineChartOptions } from '../../presentational/TimeSeriesChart/TimeSeriesChart'
+import queryAllSurveyAnswers from '../../../helpers/query-all-survey-answers'
+import format from 'date-fns/format'
+import { useInitializeView } from '../../../helpers/Initialization'
+
+export interface SurveyAnswerChartParameters {
+    label: string;
+    surveyName?: string | string[];
+    stepIdentifier?: string | string[];
+    resultIdentifier: string | string[];
+}
+
+export interface SurveyDataChartProps {
+    title?: string
+    intervalType?: "Week" | "Month" | "SixMonth"
+    weekStartsOn?: WeekStartsOn
+    charts: SurveyAnswerChartParameters[],
+    chartType: "Line" | "Bar" | "Area"
+    options?: LineChartOptions | BarChartOptions | AreaChartOptions,
+    expectedDataInterval?: Duration,
+    previewDataProvider?: (startDate: Date, endDate: Date) => Promise<SurveyAnswer[][]>
+    innerRef?: React.Ref<HTMLDivElement>
+}
+
+function getDefaultIntervalStart(intervalType: "Week" | "Month" | "SixMonth", weekStartsOn?: WeekStartsOn) {
+    let intervalStart: Date;
+    if (intervalType === "Week") {
+        intervalStart = getWeekStart(weekStartsOn);
+    } else {
+        intervalStart = getMonthStart();
+    }
+    return intervalStart;
+}
+
+export default function SurveyDataChart(props:SurveyDataChartProps) {
```

Comment:
I think I called it this because the specific values I was graphing were things that the survey *auto calculated* for you, so somewhere in my brain I concluded "These aren't technically *answers* to the survey."  But, I agree, from a technical perspective this gets data from the Survey Answer API, so it will be more clear if it's named that way.